IsEngine3.txt

```diff
@@ -0,0 +1,19 @@
+[Section]
+Name = IsEngine3 
+Version = 3.0.117.0 (file properties)
+License = Uknown
+Description = InstallSheild Engine 3.0 
+Size = 662KB
+Category = 14
+URLSite = http://toastytech.com/files/setup.html
+URLDownload Installer = toastytech.com/files/Is3Engine.zip
```

Comment:
Wrong parameter.

Reply:
> Wrong parameter.

Is because category? 
Must be = 15 (Other) ?

Must size be in MB?